deploy/008-configure-acm-commands-aggregator.ts

```diff
@@ -0,0 +1,691 @@
+  ];
+};
+
+const getConverterPermissions = (): string[][] => {
+  return [
+    accounts.flatMap(timelock => [ethers.constants.AddressZero, "pauseConversion()", timelock]),
+    accounts.flatMap(timelock => [ethers.constants.AddressZero, "resumeConversion()", timelock]),
+    accounts.flatMap(timelock => [ethers.constants.AddressZero, "setMinAmountToConvert(uint256)", timelock]),
+    accounts.flatMap(timelock => [
+      ethers.constants.AddressZero,
+      "setConversionConfig(address,address,ConversionConfig)",
+      timelock,
+    ]),
+  ];
+};
+
+const getXVSVaultTreasuryPermissions = (xvsVaultTreasury: string): string[][] => {
+  return [accounts.flatMap(timelock => [xvsVaultTreasury, "fundXVSVault(uint256)", timelock])];
+};
+
+const getPrimeRevokePermissions = (prime: string, guardian: string): string[][] => {
+  return [
+    [prime, "updateAlpha(uint128,uint128)", guardian],
+    [prime, "updateMultipliers(address,uint256,uint256)", guardian],
+    [prime, "setStakedAt(address[],uint256[])", guardian],
+    [prime, "addMarket(address,address,uint256,uint256)", guardian],
+    [prime, "setLimit(uint256,uint256)", guardian],
+    [prime, "setMaxLoopsLimit(uint256)", guardian],
+    [prime, "issue(bool,address[])", guardian],
+    [prime, "burn(address)", guardian],
+  ];
+};
+
+const getPrimeLiquidityProviderRevokePermissions = (primeLiquidityProvider: string, guardian: string): string[][] => {
+  return [
+    [primeLiquidityProvider, "setTokensDistributionSpeed(address[],uint256[])", guardian],
+    [primeLiquidityProvider, "setMaxTokensDistributionSpeed(address[],uint256[])", guardian],
+    [primeLiquidityProvider, "setMaxLoopsLimit(uint256)", guardian],
+  ];
+};
+
+const getResilientOracleRevokePermissions = (resilientOracle: string, guardian: string): string[][] => {
+  return [
+    [resilientOracle, "setOracle(address,address,uint8)", guardian],
+    [resilientOracle, "enableOracle(address,uint8,bool)", guardian],
+  ];
+};
+
+const getBoundValidatorRevokePermissions = (boundValidator: string, guardian: string): string[][] => {
+  return [[boundValidator, "setValidateConfig(ValidateConfig)", guardian]];
+};
+
+const getXVSVaultRevokePermissions = (xvsVault: string, guardian: string): string[][] => {
+  return [
+    [xvsVault, "add(address,uint256,address,uint256,uint256)", guardian],
+    [xvsVault, "set(address,uint256,uint256)", guardian],
+    [xvsVault, "setRewardAmountPerBlockOrSecond(address,uint256)", guardian],
+    [xvsVault, "setWithdrawalLockingPeriod(address,uint256,uint256)", guardian],
+  ];
+};
+
+const getRewardDistributorRevokePermissions = (guardian: string): string[][] => {
+  return [
+    [ethers.constants.AddressZero, "setLastRewardingBlock(address[],uint32[],uint32[])", guardian],
+    [ethers.constants.AddressZero, "setLastRewardingBlocks(address[],uint32[],uint32[])", guardian],
+    [ethers.constants.AddressZero, "setLastRewardingBlockTimestamps(address[],uint256[],uint256[])", guardian],
```

Comment:
We don't have `setLastRewardingBlockTimestamps` permission on Ethereum and opbnb